src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Authentication.cs

```diff
@@ -136,7 +136,9 @@ public async Task HttpClientHandler_IncorrectCredentials_Fails(string authentica
             {
                 HttpClientHandler handler = CreateHttpClientHandler();
                 Task serverTask = server.AcceptConnectionPerformAuthenticationAndCloseAsync(authenticateHeader);
-                await TestHelper.WhenAllCompletedOrAnyFailed(CreateAndValidateRequest(handler, url, HttpStatusCode.Unauthorized, new NetworkCredential("wronguser", "wrongpassword")), serverTask);
+
+                var credentials = SecretsHelper.CreateCredentials("wronguser", "wrongpassword");
```

Comment:
The purpose of this method is to annotate that the credential are for test purpose and can be part of source code.